salt/metalk8s/defaults.yaml

```diff
@@ -84,9 +84,18 @@ networks:
       127.0.0.1:7443:
         expected: nginx
         description: Apiserver proxy
+      control_plane_ip:7472:
+        expected: speaker
+        description: >-
+          Control plane MetalLB speaker metrics (only if MetalLB enabled)
+      control_plane_ip:7946:
+        expected: speaker
+        description: >-
+          Control plane MetalLB speaker (only if MetalLB enabled)
       ingress_control_plane_ip:8443:
         expected: kube-proxy
-        description: Control plane nginx ingress
+        description: >-
+          Control plane nginx ingress (only if MetalLB disabled)
```

Comment:
Don't we always need this port to be open (even if MetalLB is enabled)?

Reply:
Indeed good catch :+1: